[2025-06-04T10:33:12.752+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-04T10:33:12.775+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-04T10:33:09.398093+00:00 [queued]>
[2025-06-04T10:33:12.785+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-04T10:33:09.398093+00:00 [queued]>
[2025-06-04T10:33:12.786+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-04T10:33:12.799+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_excel_data_to_postgres> on 2025-06-04 10:33:09.398093+00:00
[2025-06-04T10:33:12.803+0000] {standard_task_runner.py:63} INFO - Started process 42 to run task
[2025-06-04T10:33:12.806+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'excel_to_postgres_test_data_dag', 'load_excel_data_to_postgres', 'manual__2025-06-04T10:33:09.398093+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/excel_to_postgres_test_data_dag.py', '--cfg-path', '/tmp/tmphcofuyj0']
[2025-06-04T10:33:12.808+0000] {standard_task_runner.py:91} INFO - Job 153: Subtask load_excel_data_to_postgres
[2025-06-04T10:33:12.854+0000] {task_command.py:426} INFO - Running <TaskInstance: excel_to_postgres_test_data_dag.load_excel_data_to_postgres manual__2025-06-04T10:33:09.398093+00:00 [running]> on host 70d97023b80f
[2025-06-04T10:33:13.384+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='excel_to_postgres_test_data_dag' AIRFLOW_CTX_TASK_ID='load_excel_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-04T10:33:09.398093+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-04T10:33:09.398093+00:00'
[2025-06-04T10:33:13.385+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-04T10:33:13.575+0000] {base.py:84} INFO - Using connection ID 'datalake_***' for task execution.
[2025-06-04T10:33:13.647+0000] {logging_mixin.py:188} INFO - Đã chèn 52 dòng dữ liệu vào test_***.test_data_tara.
[2025-06-04T10:33:13.647+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-04T10:33:13.648+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-04T10:33:13.657+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=excel_to_postgres_test_data_dag, task_id=load_excel_data_to_postgres, run_id=manual__2025-06-04T10:33:09.398093+00:00, execution_date=20250604T103309, start_date=20250604T103312, end_date=20250604T103313
[2025-06-04T10:33:13.702+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-04T10:33:13.717+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-04T10:33:13.720+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
